src/fhir/api.test.ts

```diff
@@ -3,12 +3,129 @@ import { createBundle } from "./api";
 import { initialValues, initialWithNoVariant } from "../components/reports/FormDefaults";
 import { Observation } from "fhir/r4";
 import { geneCoding } from "../code_systems/hgnc";
+import { Bundle } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/bundle";
+import { BundleEntry } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/bundleEntry";
+import { Patient } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/models-r4";
 
 const fhir = new Fhir();
 
 const reportedGenes = [geneCoding("HGNC:4389", "GNA01")];
+const FHIR_URL = process.env.REACT_APP_FHIR_URL || "";
+
+const check = async (response: Response) => {
+  const r = await response.json();
+  if (!response.ok) {
+    console.error(r.body);
+    throw new Error(response.statusText);
+  }
+  console.debug(`check response: ${JSON.stringify(r, null, 2)}`);
```

Comment:
Okay so so what we should do is to take the 
`r.entry` and map it to get the entry.response and collect any responses which start with `4` (e.g. 404 not found).
If there are any items in this list then we should throw an exception with the first outcome.issue.diagnostic.

You can use the debugger to look at the data structure and ensure your mapping the data correctly